pkgs/development/tools/tracy/default.nix

```diff
@@ -1,25 +1,29 @@
 { stdenv, lib, darwin, fetchFromGitHub
-, tbb, gtk3, glfw, pkg-config, freetype, Carbon, AppKit, capstone, dbus
+, tbb, glfw, pkg-config, freetype, Carbon, AppKit, capstone, dbus
 }:
 
 let
   disableLTO = stdenv.cc.isClang && stdenv.isDarwin;  # workaround issue #19098
 in stdenv.mkDerivation rec {
   pname = "tracy";
-  version = "0.9";
+  version = "0.9.1";
 
   src = fetchFromGitHub {
     owner = "wolfpld";
     repo = "tracy";
     rev = "v${version}";
-    sha256 = "sha256-cdVkY1dSag37JdbsoJp2/0QHO5G+zsftqwBVqRpMiew=";
+    sha256 = "sha256-K1lQNRS8+ju9HyKNVXtHqslrPWcPgazzTitvwkIO3P4";
   };
 
+  patches = [ ]
+    ++ lib.optionals (stdenv.isDarwin && !(lib.versionAtLeast stdenv.hostPlatform.darwinMinVersion "11")) [ ./0001-remove-unifiedtypeidentifiers-framework ];
+
   nativeBuildInputs = [ pkg-config ];
 
   buildInputs = [ glfw capstone ]
     ++ lib.optionals stdenv.isDarwin [ Carbon AppKit freetype ]
-    ++ lib.optionals stdenv.isLinux [ gtk3 tbb dbus ];
+    ++ lib.optionals (stdenv.isDarwin && lib.versionAtLeast stdenv.hostPlatform.darwinMinVersion "11") [ darwin.apple_sdk.frameworks.UniformTypeIdentifiers ]
+    ++ lib.optionals stdenv.isLinux [ tbb dbus pkgs.hicolor-icon-theme freetype ];
```

Comment:
`pkgs.hicolor-icon-theme` doesn't look right. Anyway, I can fix this for you tomorrow or next Monday.

Reply:
Ah yeah, pkgs is not there :)

Thanks! Let's see if it works and if not, that would be great!

Reply:
I upgraded directly to 0.9.1 so the patch is compatible.